static/js/components/form_components/buttons.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Component for rendering a button
+ */
+
+import React, { useState } from "react";
+import { styled } from "styled-components";
+
+import { saveToFile } from "../../shared/util";
+
+/* Base Button Component*/
+
+const StyledButton = styled.button`
+  align-items: center;
+  background: transparent;
+  border: 1px solid #747775;
+  border-radius: 100px;
+  display: flex;
+  font-size: 14px;
+  font-weight: 500;
+  gap: 8px;
+  justify-content: center;
+  line-height: 20px;
+  padding: 10px 24px;
+  text-align: center;
+  width: fit-content;
+
+  .icon {
+    font-size: 18px;
+  }
+`;
+
+interface ButtonProps {
+  // Class name to add to button
+  class?: string;
+  // Icon to show on button to the left of text
+  icon?: string;
+  // Text to show on button
+  label: string;
+  // Handler for what happens when button is clicked
+  onClick: () => void;
+}
+
+function Button(props: ButtonProps): JSX.Element {
+  return (
+    <StyledButton
+      onClick={props.onClick}
+      className={`button ${props.class || ""}`}
+    >
+      {props.icon && (
+        <span className="material-symbols-outlined icon">{props.icon}</span>
```

Comment:
When i load the page, the icon's name shows up as plain text for a split second before being replaced by the icon itself. is there any way to avoid showing the icon's plain text name?